Source/items.cpp

```diff
@@ -4191,38 +4191,26 @@ void __fastcall BubbleSwapItem(ItemStruct *a, ItemStruct *b)
 
 void __cdecl SortSmith()
 {
-	int v0;        // esi
-	int *v1;       // eax
-	signed int v2; // ecx
-	int *v3;       // eax
-	int v4;        // ebx
-	int v5;        // edi
+	int j, k;
+	BOOL sorted;
 
-	v0 = 0;
-	if (smithitem[1]._itype != -1) {
-		v1 = &smithitem[1]._itype;
-		do {
-			v1 += 92;
-			++v0;
-		} while (*v1 != -1);
+	j = 0;
+	while (smithitem[j + 1]._itype != -1) {
+		j++;
 	}

```

Comment:
Are the brackets requited by the style around a single `j++` line?

Reply:
No, but it's also not prohibited

Reply:
It's personal preference. I prefer always using brackets, simply because hunting down bugs caused by this is just not fun..